src/components/figures/chart/chart_dashboard_menu/chart_dashboard_menu_store.ts

```diff
@@ -21,10 +22,10 @@ export class ChartDashboardMenuStore extends SpreadsheetStore {
       const item = chartSubtypeRegistry.get(type);
       return {
         id: item.chartType,
-        label: item.displayName,
+        label: _t("Show as %s chart", item.displayName.toLowerCase()),
```

Comment:
I feel like this string alone might be difficult to translate without any context and without knowing what's going to be inserting for `%s`.
We can help translators by adding a bit of context like this
```suggestion
        label: _t("Show as %(chart_type)s chart", { chart_type: item.displayName.toLowerCase() }),
```